drivers/hud_spm_report/app/models/hud_spm_report/fy2024/episode.rb

```diff
@@ -89,8 +103,14 @@ def compute_episode(enrollments, included_project_types:, excluded_project_types
         # when not including 3.917
         []
       end
+      @debugger&.log_timeline('pre_entry_dates', pre_entry_dates)
       # note, at this point the array is no-longer sorted
       filtered_bed_nights += pre_entry_dates
+      @debugger&.log_timeline('filtered_bed_nights + pre_entry_dates', filtered_bed_nights)
+
+      filtered_bed_nights = filter_episode(filtered_bed_nights)
```

Comment:
Missing additional filter step as described in step 6, a, vi and step 6, b, vi: "Apply the same logic as described in step 2 above which may negate some nights that would otherwise be included"